Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java

```diff
@@ -134,46 +199,79 @@ public void initDefaultCommand() {
 		setDefaultCommand(new TeleopDrive());
```

Comment:
can you move the default command to the top por favor so it's easy to find and not buried in here?